languages/html.md

```diff
@@ -1,18 +1,176 @@
 ### Barrel Development Best Practices
 
+  
+
 # HTML Best Practices
 
+  
+
 ## Semantics and Readability
-Be as semantic as possible when structuring the markup of a page. Semantic markup means that tags are chosen based on the type of content and their hierarchy. These are documents and as such should read like a document when the presentation layer is stripped away. Semantic markup also makes it easier to implment accessibility standards when required. 
+
+Be as semantic as possible when structuring the markup of a page. Semantic markup means that tags are chosen based on the type of content and their hierarchy. These are documents and as such should read like a document when the presentation layer is stripped away. Semantic markup also makes it easier to implment accessibility standards when required.
```

Comment:
There's a typo for "implment", Semantic markup also makes it easier to **implment** accessibility standards when required.